src/vm/finalizerthread.cpp

```diff
@@ -22,6 +22,14 @@
 
 BOOL FinalizerThread::fRunFinalizersOnUnload = FALSE;
 BOOL FinalizerThread::fQuitFinalizer = FALSE;
+
+#ifdef FEATURE_PAL
+extern bool s_forcedGCInProgress;
+ULONGLONG FinalizerThread::LastHeapDumpTime = 0;
```

Comment:
It is actually ok here - the VM is full of windows types and we try to keep the style used in the surrounding code.